src/sdk/__mocks__/http.ts

```diff
@@ -71,6 +123,14 @@ export class Http {
 
   async get(url: string, config?: AxiosRequestConfig): Promise<AxiosResponse> {
     switch (true) {
+      case /always\/unauthorized/.test(url):
```

Comment:
I thought we agreed to remove these because they doesn't actually test any of our code, it just exercises mocks